### PR TITLE
Fix redirection for OSX

### PR DESCRIPTION
**Pull request by pepa65**
The idiom `&>` for `> 2>&1` was only introduced in bash-4.0-alpha (OSX is still on bash-3), and is deprecated in any case.

### CODE REVIEW

File: index.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 #   Homepage: https://caddyserver.com
 #   Issues:   https://github.com/caddyserver/getcaddy.com/issues
-#   Requires: bash, curl or wget, tar or unzip
+#   Requires: bash, coreutils, curl or wget, tar or unzip
```
Comment (mholt) [inline index.txt]: What needs coreutils?

Comment (pepa65) [reply index.txt]: It depends a bit on the packaging and what things are called of course. On my machine, `rm`, `type` and `mv` are part of coreutils.

Reviewing all this, `setcap` is not part of coreutils, so should be added separately. (On my system, it's part of `libcap2-bin` but I don't know how universal that is...)

I would propose to switch to just mentioning the names of all the external binaries that are required: `curl`/`wget`, `rm`, `cp`, `type`, `setcap` (not required, but will be used when present)

Comment (mholt) [reply index.txt]: That sounds better, we can probably do that.

Comment (pepa65) [reply index.txt]: I've issued a new pull request.

File: index.txt [hunk 2]
```diff
@@ -77,32 +75,33 @@ install_caddy()
...
 	else
-		echo "unsupported or unknown architecture: $unamem"
-		return 1
+		echo "Aborted, unsupported or unknown architecture: $unamem"
+		return 2
 	fi
 
 	uname="$(uname)"
-	unameu=$(tr '[:lower:]' '[:upper:]' <<<"$uname")
+	declare -u unameu=$uname
```
Comment (mholt) [inline index.txt:83]: What does -u do? I can't find it in my googling.

Comment (pepa65) [reply index.txt:83]: `help declare`: `-u        to convert NAMEs to upper case on assignment`
Bash has an amazing amount of stuff built in, regex, etc., but not always in an easily recognizable form.
If we do this, we don't need the external `tr` utility.

Comment (mholt) [reply index.txt:83]: Oh, cool. I wonder why none of the online man pages had that :P

**Review by mholt — APPROVED** [review]

**Comment by mholt** [comment]
Great -- thank you!

File: index.txt (hunk 2 above)
Comment (pepa65) [reply index.txt:83]: It is also in the bash man page, under declare/typeset.

Comment (pepa65) [reply index.txt:83]: And bash is better than tr, try this:
`echo σιγμας | tr '[:lower:]' '[:upper:]'`
And this:
`declare -u w=σιγμας; echo $w`

**Pull request merged**